app/Helpers/database/user.php

```diff
@@ -341,7 +342,7 @@ function GetDeveloperStatsFull(int $count, int $offset = 0, int $sortBy = 0, int
               INNER JOIN UserAccounts as ua ON ua.ID = tick.resolver_id
               INNER JOIN Achievements as ach ON ach.ID = tick.AchievementID
               WHERE tick.resolver_id != tick.reporter_id
-              AND ach.Author != ua.User
+              AND ach.user_id != ua.ID
```

Comment:
I think this `ua` could be eliminated too.

Reply:
This was one a bit trickier, but now is also eliminated in latest